pom.xml

```diff
@@ -562,6 +598,10 @@
             <groupId>com.github.npathai</groupId>
             <artifactId>hamcrest-optional</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Do we actually use this dependency anywhere other than tests?

Reply:
Correct, it is only for tests and the management section is defining the default scope for it to be `test`